frontends/infinite-corridor/src/components/LearningResourceCard.spec.tsx

```diff
@@ -67,11 +59,19 @@ describe("LearningResourceCard", () => {
 
   test.each([
     { isAuthenticated: true, canAddToList: true },
-    { isAuthenticated: false, canAddToList: false }
+    { isAuthenticated: false, canAddToList: false },
+    {
+      isAuthenticated: false,
+      canAddToList:    false,
+      resource:        {
+        ...makeSearchResult()._source,
+        lists: undefined
+      }
+    }
```

Comment:
This is the new test case. All other changes in this file are just because I simplified the signature of `setup()` a bit.